src/main/java/com/kainos/discoverydiary/models/Media.java

```diff
@@ -24,6 +26,20 @@ public Media(String title, String author, String description, Category category,
         this.description = description;
         this.category = category;
         this.publicationDate = publicationDate;
+        this.status = Status.AVAILABLE;
+    }
+
+    public Media(String title, String author, String description, Category category, String publicationDate, String nameOfBorrower, Status status) {
```

Comment:
Could you make a private constructor which constructs the common things?
        this.id = count;
        this.title = title;
        this.author = author;
        this.description = description;
        this.category = category;
        this.publicationDate = publicationDate